README.md

```diff
@@ -1,87 +1,159 @@
+go into a message).
+* **How**: if multiple approaches for achieving your goal were available, you also want to explain _why_ you chose the
+used implementation strategy.
     - Note that you should not explain how your change achieves your goal in your commit message.
       That should be obvious from the code itself.
       If you cannot achieve that clarity with the used programming language, use comments within the code instead.
     - The commit message is primarily the place for documenting the _why_.
 
-Unfortunately, checking whether these last questions get answered is also the most difficult to do automatically.
-This tool only checks for a few common errors other than syntax errors:
-1. Usage of Jira ticket numbers in the subject line.
-    - Because the subject line is expensive real estate every character should be most efficiently used to convey meaning to humans.
-    - Jira ticket numbers are not equal to the tickets themselves and thus convey very little information.
-      These ticket numbers should go in message footers where tools can still extract them for automatic linking.
-2. Using non-imperative verb forms (adds, added or adding instead of add) in the subject line.
-    - These other forms convey no more meaning but use extra precious characters.
-3. Referring to review comments that cannot be found anywhere in the commit history itself.
-    - Commit messages should be self-contained.
-      Only mentioning that a commit is created in response to a review comment, without mentioning the reasoning of that comment is clearly not self-contained.
-    - Whenever your workflow permits it, prefer amending the original commit (or using `--fixup`) instead.
-    - If your workflow doesn't permit that, or it seems suboptimal in a given context, describe what, why and how you are changing (as mentioned before).
 
 ## Installation
 
 You can install this package with pip as follows:
 
 ```sh
-pip3 install --user --upgrade commisery
+$ pip3 install --user --upgrade commisery
 ```
 
 ## Usage
 
-You can verify commit messages with the included CLI tool:
+Basic usage instructions:
+
+```
+Usage: cm [OPTIONS] COMMAND [ARGS]...
+
+  Manages conventional commit messages
+
+Options:
+  -c, --config TEXT               Path towards a configuration file
+  -t, --tags TEXT                 Comma-separated list of accepted
+                                  conventional commit tags to allow aside from
+                                  the default "feat" and "fix". If omitted,
+                                  uses the following list: fix, feat, build,
+                                  chore, ci, docs, perf, refactor, revert,
+                                  style, test, improvement
+
+  -l, --max-subject-length INTEGER
+                                  Maximum characters allowed in the subject of
+                                  the commit message
+
+  -d, --disable TEXT              List of commit message rules to disable. Can
```

Comment:
Maybe add also reference to the location where details of the rules can be found?

Reply:
The details can be request with `cm overview`, including a description of each rule.